NuGet.config

```diff
@@ -5,6 +5,7 @@
   </fallbackPackageFolders>
   <packageSources>
     <clear />
+    <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
```

Comment:
Should this feed get checked in?

Reply:
Yes it should - as long at the checked in packages come from this feed it should be checked in to support regen.  This should get cleaned up once the preview maintenance packages are removed and the feed is no longer required.

Reply:
The generate.sh script fails to find the packages without this feed.